libs/client-sdk/src/data_sources/data_source.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
```

Comment:
Copying this from `internal` lib, as it will be removed when the server-side SDK is introduced and we don't want to port over the client immediately.